.github/configs/typos.toml

```diff
@@ -0,0 +1,10 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
+[files]
+extend-exclude = [
+    "*.svg",
+    "*.drawio",
+]
+[default.extend-words]
+BA = "BA"
+Ded = "Ded"
+EDE = "EDE"
```

Comment:
What do these lines mean? I quickly scanned the suggested reference documents, but still couldn't understand this meaning quickly.

Reply:
`[files]` list the files that should be filtered not to scan.

`[default.extend-words]` list the black list of words that should be filtered not to scan.

Reply:
> What do these lines mean?

simply means ignore `BA`、`Ded`、`EDE` these words.